cmd/bolt-agent/main.go

```diff
@@ -228,6 +228,11 @@ func getApp() *cli.App {
 			Hidden: true,
 			Value:  5 * time.Minute,
 		},
+		&cli.BoolFlag{
```

Comment:
smooth is weird name. Where is this used ?

Reply:
It's not really something new, just wasn't exposed before. Yeah, I'd also like to come up with a better name 😅 . Idea of `smooth` is to "smooth out" HTLCs. That is if you forward one your channel balance won't just drop and if it fails raise back again, but just take the final state into consideration. It's a hack pretty much that's why it is off by default.